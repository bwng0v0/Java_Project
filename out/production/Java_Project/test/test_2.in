1
2
p 1234567890 55.5 true :41-1, Daechi-dong, Gangnam-gu, Seoul:
0
0
