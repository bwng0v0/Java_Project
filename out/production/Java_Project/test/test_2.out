
******** Main Menu ********
* 0.exit 1.JavaBasic(ch2) *
***************************
menu item? 1

************* Ch2 Menu ***********
* 0.exit 1.output 2.readToken    *
* 3.readLine 4.operator 5.switch *
**********************************
menu item? 2
person information(name id weight married :address:):
p 1234567890 55.5 true :41-1, Daechi-dong, Gangnam-gu, Seoul:

************* Ch2 Menu ***********
* 0.exit 1.output 2.readToken    *
* 3.readLine 4.operator 5.switch *
**********************************
menu item? 0

******** Main Menu ********
* 0.exit 1.JavaBasic(ch2) *
***************************
menu item? 0

Good bye!!
